src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/BrotliStream.cs

```diff
@@ -61,8 +61,7 @@ public BrotliStream(Stream stream, CompressionMode mode, bool leaveOpen)
 
         private void EnsureNotDisposed()
```

Comment:
```suggestion
        [MemberNotNull(nameof(_stream))]
        private void EnsureNotDisposed()
```

Reply:
These annotations weren't there previously.  These changes don't change the need for them.  The only benefit to such annotation would be if someone needed to know that _stream is non-nullable after the Ensure call, and that doesn't exist today.  Such annotation can be added to internal/private members as needed.

Reply:
The changes introduce a method call. I believe `MemberNotNullAttribute` may provide a benefit similar to `DoesNotReturnAttribute`, as code analysis can determine the method will not return when the member is null.